src/js/descript.js

```diff
@@ -135,5 +167,41 @@
         return containerName ? this._containers[containerName] : this._containers;
     };
 
+    /**
+     * Replaces string patterns in inline scripts. Patterns can be
+     * @param containerName
```

Comment:
We need the `containerName` here to specify where you're wanting to replace scripts, so I think this stays.